tests/src/run.sh

```diff
@@ -28,6 +28,6 @@ mkdir -p ./logs
 
 export LD_LIBRARY_PATH=/usr/lib:/lib:/home/root:${MY_DIR}
 
-./ut-test -l ./logs -p ./assets/test_kvp.yaml $@
+./ut-test -a -l ./logs -p ./assets/test_kvp.yaml $@
```

Comment:
that's not correct and passed via `$@`

-a is optional, basic mode needs to be supported on both.